activemodel/lib/active_model/type/value.rb

```diff
@@ -129,6 +129,10 @@ def hash
       def assert_valid_value(_)
       end
 
+      def wrap_value(value)
```

Comment:
Should it be `# :nodoc`?

Reply:
Yep.  Just updated the method name and set to `:nodoc:`